### PR TITLE
README.md and FUNDING.yml added

### LINKED ISSUE #1: Create a GitHub landing page
Create a full-fledge GitHub landing page for the tapaScript GitHub.

### PR DESCRIPTION
**Pull request by anshumanSathua**
Issue id : #1 
Changes proposed :-
> Added the README.md file for the Repo
> Added the FUNDING.yml file for the Sponsoring

 - [x] Logo
 - [x] All Socials (youtube, website, twitter, discord, email)
 - [x] A bit about tapaScript
 - [x] Who should join tapaScript?
 - [ ] YouTube Badge
 - [x] Embedding this video: https://www.youtube.com/watch?v=90BgcWNRFMk
 - [x] About Sponsoring

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Greetings, Developers! I am Tapas Adhikary, your guide on this exhilarating journey into the realms of UI, UX, Web Development, JavaScript, React, Next.js, Node.js, and the captivating universe of FullStack engineering. Having immersed myself in the world of code since 2000, I bring 19 years of experience in crafting production-ready software.
+
+At tapaScript, my commitment is not just to teach but to make the fundamentals an unforgettable part of your coding repertoire. Picture mastering programming with unwavering confidence - that's the essence of my approach.
+
+My coding odyssey, initiated in 2004, has been a remarkable adventure. From Open Source contributions, notably in ReactPlay, to my engagement with various platforms such as "GreenRoots Blog," FreeCodeCamp, CSS-Tricks, and more, I have honed my skills and shared my insights.
+
+Join me on YouTube, where I pour years of gathered wisdom into empowering you. Together, let's make tapaScript a place where fundamental concepts become second nature, and you emerge with the confidence to navigate the intricacies of software development.
+
+Welcome to tapaScript - where the beauty of mastering code unfolds. 👨‍💻✨
```
Comment (atapas) [inline README.md]: Can we centre this section?

image

**Review by atapas — CHANGES_REQUESTED** [review]

**Comment by atapas** [comment]
> Issue id : #1 Changes proposed :-
> 
> > Added the README.md file for the Repo
> > Added the FUNDING.yml file for the Sponsoring
> 
> * [x]  Logo
> * [x]  All Socials (youtube, website, twitter, discord, email)
> * [x]  A bit about tapaScript
> * [x]  Who should join tapaScript?
> * [ ]  YouTube Badge
> * [x]  Embedding this video: https://www.youtube.com/watch?v=90BgcWNRFMk
> * [x]  About Sponsoring

Here is the example of YouTube badge

image

Refer: https://github.com/atapas/youtube

File: README.md (hunk 1 above)
Comment (atapas) [reply README.md]: Also remove the Gmail... you can add FB if available :)

Comment (anshumanSathua) [reply README.md]: - [x] Centered that section
- [x] Removed Gmail
- [x] Added FB and WP
- [x] Added YouTube Badge

**Review by atapas — APPROVED** [review]
@anshumanSathua Looks great

**Pull request merged**